index.js

```diff
@@ -25,9 +25,12 @@ const client = new Client({
 });
 
 client.commands = new Collection();
+client.aliases = new Collection();
```

Comment:
We don't need an extra collection for aliases. I already found an easy way to use aliases, I'll add it as soon as possible

Reply:
okay i'll just remove the aliases part for now